Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1464,6 +1464,29 @@ void CClientVehicle::SetWheelStatus(unsigned char ucWheel, unsigned char ucStatu
 
                 // Update the wheel's visibility
                 m_pVehicle->SetWheelVisibility((eWheelPosition)ucWheel, (ucStatus != DT_WHEEL_MISSING));
+
+                // restart wheels component visibility (fix #1556)
+                static SString strComponentName;
+                static unsigned char ucComponentWheel;
```

Comment:
In the conversation above I meant that `"wheel_"` implicitly creates a new string every time `BeginWith` called. And it's better to create a static object for `"wheel_"`. Something like: `const static SString WHEEL_PREFIX("wheel_");`. There is not need for static variables `strComponentName `and `ucComponentWheel`.

Reply:
Okay, done!